src/utils_request.js

```diff
@@ -26,6 +26,29 @@ export const REQUEST_AS_BROWSER_DEFAULT_OPTIONS = {
     proxyUrl: null,
     timeoutSecs: 30,
 };
+
+/**
+ * @typedef {Object} RequestAsBrowserOptions
+ * @property {String} url
+ *  URL of the target endpoint. Supports both HTTP and HTTPS schemes.
+ * @property {String} [method=GET]
+ *  HTTP method.
+ * @property {Object} [headers]
+ *  Additional HTTP headers to add. It's only recommended to use this option,
+ *  with headers that are typically added by websites, such as cookies. Overriding
+ *  default browser headers will remove the masking this function provides.
+ * @property {String} [languageCode=en]
+ *  Two-letter ISO 639 language code.
+ * @property {String} [countryCode=US]
+ *  Two-letter ISO 3166 country code.
+ * @property {Boolean} [isMobile]
+ *  If `true`, the function uses User-Agent of a mobile browser.
+ * @property {Function} [abortFunction]
+ *  Function accepts `response` object as a single parameter and should return true or false.
+ *  If function returns true request gets aborted. This function is passed to the
+ *  (@apify/http-request)[https://www.npmjs.com/package/@apify/http-request] NPM package.
```

Comment:
`@property {string} [proxyUrl]` as it's accepted by underlying `httpRequest`

Reply:
Why the `null`? Wouldn't the `[proxyUrl]` suffice to define that it's optional? I'm not aware of any special functionality being triggered by `null`.

Reply:
Won't do. @pocesar This is beyond the scope of this PR. Keep notes though.

@mnmkng `null` and `undefined` are distinct and unassignable types. `{String} [proxuUrl]` translates into `proxyUrl? string` and if you try to assign `null` this happens:

https://www.typescriptlang.org/play/#code/FAFwngDgpgBAIlAZjAvDA3sG2YEMDO+ArgE5QAmAXDPiCQJYB2A5gDRY4D2AtvSCLgBGAGygB+arQYtgAXwDcwYAGNOjWjAAe1BMjSYceQqQrUARAEk83GAAsoZM+0M8+AkVGqMiw4XKA

Reply:
Yeah, I was referring to the fact that we don't need the `null` there runtime-wise.

Reply:
I'm really missing the proxyUrl here, right now it shows as "non part" of the options object
![image](https://user-images.githubusercontent.com/461084/73282568-98284380-41d0-11ea-95e7-bcde21b3bd6d.png)
when using BasicCrawler and getApifyProxyUrl, it gets in the way

Reply:
Done. Enjoy.